src/generators/legacy-json/utils/buildSection.mjs

```diff
@@ -0,0 +1,392 @@
+import { unified } from 'unified';
+import html from 'remark-html';
+import {
+  DEFAULT_EXPRESSION,
+  LEADING_HYPHEN,
+  NAME_EXPRESSION,
+  PARAM_EXPRESSION,
+  RETURN_EXPRESSION,
+  TYPE_EXPRESSION,
+} from '../constants.mjs';
+import { buildHierarchy } from './buildHierarchy.mjs';
+import parseSignature from './parseSignature.mjs';
+
+const sectionTypePlurals = {
+  module: 'modules',
+  misc: 'miscs',
+  class: 'classes',
+  method: 'methods',
+  property: 'properties',
+  global: 'globals',
+  example: 'examples',
+  // Constructors should go under a class sections' signatures property
+  ctor: 'signatures',
+  classMethod: 'classMethods',
+  event: 'events',
+  var: 'vars',
+};
+
+/**
+ * @param {import('../types.d.ts').HierarchizedEntry} entry
+ * @returns {import('../types.d.ts').Meta | undefined}
+ */
+function createMeta(entry) {
+  const makeArrayIfNotAlready = val => (Array.isArray(val) ? val : [val]);
+
+  const { added_in, n_api_version, deprecated_in, removed_in, changes } = entry;
+  return {
+    changes,
+    added: added_in ? makeArrayIfNotAlready(added_in) : undefined,
+    napiVersion: n_api_version
+      ? makeArrayIfNotAlready(n_api_version)
+      : undefined,
+    deprecated: deprecated_in
+      ? makeArrayIfNotAlready(deprecated_in)
+      : undefined,
+    removed: removed_in ? makeArrayIfNotAlready(removed_in) : undefined,
+  };
+}
```

Comment:
```suggestion
function createMeta(entry) {
  const makeArrayIfNotAlready = val => (Array.isArray(val) ? val : [val]);

  const { added_in, n_api_version, deprecated_in, removed_in, changes } = entry;
  if (!added_in && !n_api_version && !deprecated_in && !removed_in && changes.length < 1) {
    return undefined;
  }

  return {
    changes: changes.length > 0 ? changes : undefined,
    added: added_in ? makeArrayIfNotAlready(added_in) : undefined,
    napiVersion: n_api_version
      ? makeArrayIfNotAlready(n_api_version)
      : undefined,
    deprecated: deprecated_in
      ? makeArrayIfNotAlready(deprecated_in)
      : undefined,
    removed: removed_in ? makeArrayIfNotAlready(removed_in) : undefined,
  };
}
```

In the old generator, if there was no metadata, the property was emitted entirely.